bun/src/binding.ts

```diff
@@ -116,10 +113,13 @@ const lib = dlopen(binaryPath, {
 		],
 		returns: "u32",
 	},
-});
+};
 
+// Load the compiled Zig shared library
+const lib = await openLibrary(path.resolve("../bun"), fns);
```

Comment:
the `path.resolve("../bun")` is strange here. I don't think it will work when the cwd is another consumer of this package.

maybe better is `path.join(import.meta.dirname, '..')`

Reply:
I simplified it to `const lib = await openLibrary(path.resolve("."), fns);` please have a look at the latest change

Reply:
@twoeths check this comment. Try something with import.meta.dirname here instead of path.resolve/cwd

Reply:
thanks, just switched to `"import.meta.dirname"`